src/config.js

```diff
@@ -0,0 +1,54 @@
+export const configuration = {
+  form: {
+    baseElement: window,
+    queryElement: document,
+    formioConfig: {
+      showCancel: false,
+      showPrevious: false,
+      showNext: false,
+      showSubmit: false,
+    },
+    adminEmail: '',
+    endpoint: 'submission',
+    pdfEndpoint: 'kyfbpdf',
```

Comment:
when will this be multi environment?

Reply:
I don't follow.

Reply:
the pdfEndpoint, should it be next do the questionnaire endpoint for the different environments?

Reply:
@jordanbinskin doesn't the staging keep this the same?